config/routes.rb

```diff
@@ -5,7 +5,9 @@
 
   resources :summaries, :fines, :checkouts, only: [:index]
   resources :renewals, only: [:create]
-  resources :holds, only: [:index, :new, :update, :destroy]
+  resources :holds, only: [:index, :new, :create, :update, :destroy]
+
+  match 'holds/result' => 'holds#result', via: :get, as: :result
```

Comment:
```suggestion
  get 'holds/result', to: 'holds#result', as: :result
```
> In general, you should use the get, post, put, patch and delete methods to constrain a route to a particular verb
https://guides.rubyonrails.org/routing.html#http-verb-constraints